iroh-relay/src/server.rs

```diff
@@ -1106,4 +1143,58 @@ mod tests {
         assert_eq!(txid, txid_back);
         assert_eq!(response_addr, socket.local_addr().unwrap());
     }
+
+    #[tokio::test]
+    async fn test_relay_access_reject() {
+        let _guard = iroh_test::logging::setup();
+
+        let a_secret_key = SecretKey::generate(rand::thread_rng());
+        let a_key = a_secret_key.public();
+
+        let server = Server::spawn(ServerConfig::<(), ()> {
+            relay: Some(RelayConfig::<(), ()> {
+                http_bind_addr: (Ipv4Addr::LOCALHOST, 0).into(),
+                tls: None,
+                limits: Default::default(),
+                key_cache_capacity: Some(1024),
+                access: AccessConfig::Restricted(Box::new(move |node_id| {
+                    info!("checking {}", node_id);
+                    // reject node a
+                    if node_id == a_key {
+                        Access::Deny
+                    } else {
+                        Access::Allow
```

Comment:
You're not testing this path.  I guess it's not really needed from a coverage viewpoint.  But it being written with this specific if condition rather than a blanket deny makes it stand out and seem like you only implemented half of the test.

Reply:
done